pkg/controller/migcluster/validation.go

```diff
@@ -15,32 +15,36 @@ import (
 	liberr "github.com/konveyor/controller/pkg/error"
 	migapi "github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
 	migref "github.com/konveyor/mig-controller/pkg/reference"
+	"github.com/pkg/errors"
```

Comment:
Looking at other places in the codebase, I think we just want the pkg "errors", not github.com/pkg/errors.

Reply:
I was referencing the dependency in migcluster_types.go on line 30 .. seems that I need that specific package to access errors.Errorf

Reply:
removing in favor of using GetOperatorVersion function like you suggested below :+1: